src/components/Dropdown/Dropdown.tsx

```diff
@@ -133,8 +137,8 @@ export interface DropdownProps extends UIComponentProps<DropdownProps, DropdownS
   /** Sets search query value (controlled mode). */
   searchQuery?: string
 
-  /** Whether toggle button (that shows/hides items list) should be rendered. */
-  toggleButton?: boolean
+  /** Whether a toggle indicator (that shows/hides items list) should be rendered. */
```

Comment:
seems that we should introduce some corrections to description (as it is not `boolean` anymore). Something like the following, if we would like to stay consistent with the rest of the code:

```
/* Toggle button that shows/hides items list. */
```
or more precise:

```
/* Controls appearance of toggle button that shows/hides items list. */
```